controller_interface/include/controller_interface/controller_interface.hpp

```diff
@@ -42,6 +42,14 @@ class ControllerInterface : public controller_interface::ControllerInterfaceBase
   CONTROLLER_INTERFACE_PUBLIC
   bool is_chainable() const final;
 
+  /**
+   * Controller has no state interfaces.
```

Comment:
```suggestion
   * A non-chainable controller has no state interfaces.
```
Wondering is this wording is better?

Reply:
```suggestion
   * A non-chainable controller doesn't export any state interfaces.
```